src/styles/components/Breadcrumb.scss

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ @import "../common/colors";
+ @import "../common/fonts";
+ @import "../common/mixins";
+
+ .breadcrumb1 {
+    display: block; 
+    width: fit-content;
+    padding: 23px 39px; 
+    &__breadcrumb-item-active{
```

Comment:
Instead of adding 'breadcrumb1' and out own class, can we achieve the same result using React-Bootstrap classes like below?
.breadcrumb-item {
    display: block; 
    width: fit-content;
    padding: 23px 39px;
   @include breadcrumb-item($blue-600)
   &.active {
          @include breadcrumb-item($gray-900)
    }
}